wiki/Ranking_Criteria/Visual_Content_Considerations/en.md

```diff
@@ -0,0 +1,117 @@
+
+### Swimsuits
+
+Depictions of *reasonable* swimsuits is allowed so long as said depiction is not highly excessive or in violation of the posturing and innuendo rules listed in the above section.
+
+Highly excessive is a broad term, but is generally taken to mean when an image treads well into the line of questionable content, such as when a character may be wearing a swimsuit far too small to adequately cover their physical assets, or if said physical assets command a dominating amount of the scene.
+
+Swimsuits that are too small or geared to be overtly revealing are generally not allowed if they are in good faith, considered questionable by a reasonable quantity of users.
+
+### Mild to moderate violence
+
+Scenes involving active combat, fighting or any other physical activity that may involve the presence of minor wounds or injuries s considered acceptable so long as there is not an excessive presence of gore, bloodshed, or distressing physical impairment.
+
+### Artistic nudity
+
+Any depiction of classical artwork or the rendition of the human form where the portrayal of partial nudity is done in a strictly artistic and non-sexual context may be allowed upon review from the GMT.
+
+However, said artistic nudity must possess some contextual relevance to the other content it is associated with in order for this exception to apply. 
+
+An example of proper application of this rule would be for content using the *Immersion* band cover from Pendulum, which features a naked man and woman pressing their hands together in a sidelong shot in the center of the piece. The icon is relatively small, understated, but discernible, and is considered allowed due to its non-sexual nature and contextual relevance to tracks from that album as a whole.
+
+### Romance, flirting or intimate affection
+
+Depiction of romantic acts or common intimacy between characters in an established narrative context is acceptable so long as it does not tread upon the rules listed above.
+
+This also includes "flirty" depictions which are - in good faith - considered as playful, not unreasonably excessive, and not in violation of any of the other inappropriate content markers.
+
+## Further considerations
+
+The anecdotal 'rule of thumb' is that one should generally be comfortable with their family members or other judgemental authority figure viewing the content and be capable of adequately explaining what is going on in it. 
+
+This is by no means a strict requirement, but it is a good way to help inform your decisions on what is and isn't appropriate.
+
+### What do I do if I am uncertain about a piece of visual content?
+
+In situations whether the appropriateness of a given piece of visual content is unclear, the mapper must submit a formal request to any Beatmap Nominator for further appraisal, who will then refer the issue to the Global Moderation Team to decide based on a supermajority (>70%) consensus.
```

Comment:
Why the extra step with the BNs? We all know that some of them are absolutely unable to judge visual content and asking them to ask a GMT member is just unnecessary. In my opinion (not that it matters) the mapper should just submit that formal request to the gmt.

Reply:
backgrounds will be discussed on bn discord though, so asking bns to send them in there for discussion would be quicker i believe, also not every GMT is responsible for visual content

Reply:
Then first, the wording suggests that each GMT member takes part in the discussion, not just a few. Second, what you said suggests that BNs will take part in that discussion, which is again, not part of the original paragraph if that is the case. 

And last, if the second scenario is not the case, then it will still be easier to list those GMT members and send a request to them, or have a submission form what goes directly to these members, instead of circulating the request all around osu!. Though, if they have to ask whether something is good or not, they should already consider changing it.

Reply:
I think you can also ask a GMT member directly, but since they also have different tasks to fulfill and Beatmap Nominators are already on the frontline of beatmap management and evaluation, they have been given a more direct route to the GMT to file requests here for efficiently. Sure a user can opt to ask the GMT directly, but it might be more efficient for every party involved if the BN takes over and forwards it to the right places, as they will be handling the beatmap in question anyways.